Configurations/windows-makefile.tmpl

```diff
@@ -499,12 +500,16 @@ install_runtime_libs: build_libs
 install_programs: install_runtime_libs build_programs
 	@if "$(INSTALLTOP)"=="" ( $(ECHO) "INSTALLTOP should not be empty" & exit 1 )
 	@$(ECHO) "*** Installing runtime programs"
-	@"$(PERL)" "$(SRCDIR)\util\mkdir-p.pl" "$(INSTALLTOP)\bin"
-	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_PROGRAMS) \
+	@if not "$(INSTALL_PROGRAMS)"=="" \
```

Comment:
Just one more question, for my understanding: does the ONECORE build generate only libraries and no binaries? Or asking differently: why and when is  `$(INSTALL_PROGRAMS)` empty?

Reply:
It’s orthogonal. No binaries will be generated when you specify no-apps or any other options imply that

Reply:
With some configuration settings, "apps" is disabled, which results in an empty `$(INSTALL_PROGRAMS)`, so this actually clears a possible issue that has nothing per se to do with ONECORE....  except if you build for ARM, for which we mandate `no-stdio`, which in turn disables "apps"

Reply:
Thanks.